### PR TITLE
People who have an upcoming OOH rotation should receive a notification a week in advance

### PR DESCRIPTION
**Pull request by lozette**
Trello: https://trello.com/c/fYbE2CJk/67-people-who-have-an-upcoming-ooh-rotation-should-receive-a-notification-a-week-in-advance

This application already sends a Slack notification when a user is about to start their rotation. This PR adds an email alert as well, sent via SendGrid.

### CODE REVIEW

File: lib/rota_alerts.js [hunk 1]
```diff
@@ -25,8 +26,9 @@ class RotaAlerts {
     const date = moment(startDate).format("dddd Do MMMM");
     const text = this.message(date);
     const slackMessage = new SlackMessage(email, text);
+    const emailMessage = new EmailMessage(email, text);
 
-    return await slackMessage.send();
+    return await [slackMessage.send(), emailMessage.send()];
```
Comment (pezholio) [inline lib/rota_alerts.js:31]: I think you might need to do `await Promise.all([slackMessage.send(), emailMessage.send()])` here

**Review by pezholio — CHANGES_REQUESTED** [review]
Coupla things. I reckon we should have a look at the Sendgrid option. Happy to pair on it if you want 👍

File: lib/rota_alerts.js (hunk 1 above)
Comment (lozette) [reply lib/rota_alerts.js]: I have no idea so I'll take your word for it 😆

File: lib/email_message.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  constructor(emailAddress, message) {
+    this.emailAddress = emailAddress;
+    this.message = message;
+  }
+
+  async send() {
+    const transporter = nodemailer.createTransport({
+      sendmail: true,
+      newline: "unix",
+      path: "/usr/sbin/sendmail",
```
[the remark this answers is not in the document]
Comment (lozette) [reply lib/email_message.js:13]: Yes, I have no idea about how to actually send the mail so that's do that

**Pull request merged**